config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -42,13 +43,36 @@ const aliasMonoRepoPackages = (monoRepo, hotReloading, webpackAliasOptions = {},
   }, {});
 };
 
+/**
+* Create index.html files for terra-dev-site and the additional apps.
+*/
+const indexPlugin = ({
+  title, filename, lang, rootElementId = 'root', siteConfig, indexEntryPoints, entry,
+}) => {
+  const otherIndexChunks = indexEntryPoints.filter(indexEntry => indexEntry !== entry);
+  return new HtmlWebpackPlugin({
+    title,
+    filename,
+    template: path.join(__dirname, '..', '..', 'lib', 'index.html'),
+    lang,
+    rootElementId,
```

Comment:
Do we need to create a terra webpack env to allow custom root ids? I see we provide a fallback value, but maybe I'm missing where one could customize it?

Reply:
This shouldn't be customized, it's exposed because with orion-dev-site we'll be re-using this to generate a second site.

Reply:
Are we also reusing the html template? In orion-dev-site will we be defining a different selector?